pinot-minion/src/main/java/org/apache/pinot/minion/MinionContext.java

```diff
@@ -39,7 +41,7 @@ public static MinionContext getInstance() {
 
   private File _dataDir;
   private MinionMetrics _minionMetrics;
-  private String _minionVersion;
+  private ZkHelixPropertyStore<ZNRecord> _helixPropertyStore;
```

Comment:
+1 for this!

I thought of fetching table config calling controller API. This approach is much simpler :)